exporter/elasticsearchexporter/model.go

```diff
@@ -169,123 +165,31 @@ func (m *encodeModel) encodeLogECSMode(resource pcommon.Resource, record plog.Lo
 	return document
 }
 
-func (m *encodeModel) encodeMetrics(resource pcommon.Resource, metrics pmetric.MetricSlice, _ pcommon.InstrumentationScope) ([][]byte, error) {
```

Comment:
Let's create an issue to re-introduce grouping metrics with the same attributes and data_stream.* values into the same document.

Reply:
Ah, I see you already did that: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33756